### PR TITLE
EMP-93: Updated task repository and controller for list results

### PR DESCRIPTION
**Pull request by tanovellino**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LASB-XXX)

Added entities for lookup data
Updated Tasks entity with joins
Updated Task controller to have 2 dedicated endpoints: one for fields required to list results and another for the ODF image file.

## Checklist

Before you ask people to review this PR:

- [x] Tests should be passing: `./gradlew test`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [x] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [x] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/TaskController.java [hunk 1]
```diff
@@ -10,31 +10,30 @@
...
-    private final TasksRepository tasksRepository;
+    private final TaskRepository taskRepository;
 
-    @GetMapping(value="/test")
-    public ResponseEntity<LocalDateTime> getTasks() {
-        Tasks task = tasksRepository.findById(5001483L)
+    @GetMapping(value="/{id}")
+    public ResponseEntity<Task> getTasks(@PathVariable("id") long id) {
+        Task task = taskRepository.findById(id)
                 .orElseThrow(() -> new ResourceNotFoundException("Not found Comment with id = " + 5001483L));
```
Comment (tanovellino) [inline equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/TaskController.java:27]: I will remove hard coded ID from here

**Comment by solomonmanik** [comment]
LGTM

File: equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/repository/TaskRepository.java [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
-
 import org.springframework.data.jpa.repository.JpaRepository;
 
-import uk.gov.justice.laa.crime.equinity.historicaldata.model.Tasks;
+import uk.gov.justice.laa.crime.equinity.historicaldata.model.Task;
+
 
-public interface TasksRepository extends JpaRepository<Tasks, Long> {
+public interface TaskRepository extends JpaRepository<Task, Long> {
   //  List<Tasks> findByID(Long ID);
```
Comment (solomonmanik) [inline equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/repository/TaskRepository.java:9]: Pls remove commented code

Comment (tanovellino) [reply equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/repository/TaskRepository.java:9]: I will removed on the next PR with the new changes

**Review by solomonmanik — APPROVED** [review]
LGTM

**Pull request merged**